v2/stack_analyses.yaml

```diff
@@ -0,0 +1,469 @@
+        schema:
+          type: string
+          default: vscode
+          enum:
+            - vscode
+      - name: show_transitive
+        in: header
+        description: This is required to enable or disable the transitive support.
+          We need to pass the value as 'true' if transitive is needed. Default value
+          is 'false'.
+        schema:
+          type: boolean
+          default: true
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              required:
+              - filePath[]
+              - manifest[]
+              properties:
+                manifest[]:
+                  type: string
+                  description: A manifest file dependencies.txt for maven, npmlist.json
+                    for npm & pylist.json for pypi. Commands to generate these files
+                    are mentioned in the README.md
+                  format: binary
+                filePath[]:
+                  type: string
+                  description: The absolute or relative path where the manifest file
+                    is located. This has to be mandatorily provided with the manifest[]
+                    parameter and vice versa.
+        required: true
+      responses:
+        200:
+          description: Response with the unique identifier for the stack analysis
+            request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserStackIdentifier'
+        400:
+          description: Bad request
+          content: {}
+        401:
+          description: Request Unauthorized
+          content: {}
+  /stack-analyses/{request-id}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about an application stack and related
+        recommendations to improve the stack
+      parameters:
+      - name: request-id
+        in: path
+        description: stack analysis request id
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StackAnalysesReport'
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Data not found
+          content: {}
+        408:
+          description: Request has timed out on server
+          content: {}
+components:
+  schemas:
+    VulnerabilityForRegisteredUser:
+      title: VulnerabilityForRegisteredUser
+      allOf:
+        - $ref: '#/components/schemas/ExtendedVulnerabilityFields'
+        - oneOf:
+          - $ref: '#/components/schemas/PublicVulnerability'
+          - $ref: '#/components/schemas/PrivateVulnerability'
+      description: Vulnerability details for registered users
+    VulnerabilityForFreeTier:
```

Comment:
Please make this plural 'Vulnerabilities'

Reply:
It holds **single** vulnerability details, it is not array of vulnerabilities.